bentoml/service/inference_api.py

```diff
@@ -298,6 +298,8 @@ def handle_request(self, request: flask.Request):
         result = next(iter(results))
         response = self.output_adapter.to_http_response(result)
         response.headers['X-Request-Id'] = inf_task.task_id
+        if self.output_adapter.cors:
+            response.headers["Access-Control-Allow-Origin"] = self.output_adapter.cors
```

Comment:
could you also add a unit test case to cover this and the `handle_batch_request` below?